bitsail-connectors/bitsail-connectors-legacy/bitsail-connector-hudi/src/test/java/com/bytedance/bitsail/connector/legacy/hudi/Fake2HudiITCase.java

```diff
@@ -39,7 +39,7 @@
 public class Fake2HudiITCase {
   private static final Logger LOG = LoggerFactory.getLogger(Fake2HudiITCase.class);
   private static final String TEST_SCHEMA =
-      "[{\"name\":\"id\",\"type\":\"string\"},{\"name\":\"text\",\"type\":\"string\"},{\"name\":\"timestamp\",\"type\":\"string\"}]";
+      "[{\"name\":\"id\",\"type\":\"string\"},{\"name\":\"text\",\"type\":\"string\"},{\"name\":\"age\",\"type\":\"byte\"},{\"name\":\"timestamp\",\"type\":\"string\"}]";
```

Comment:
Actually the ITCase test will fail because `byte` type is not supported in `com.bytedance.bitsail.connector.legacy.fake.source.FakeSource`.
So I suggest focusing on `FakeSource` because it is a basic connenctor and widely used in various tests, and the modification to hudi connector can be ignored.

Reply:
Noted, will update PR accordingly